.github/workflows/coding-style.yml

```
@@ -0,0 +1,54 @@
+      - v*
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  php-cs-fixer:
+    name: Php-Cs-Fixer
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)